src/components/ui/BaseDropdown.vue

```diff
@@ -42,6 +43,7 @@
 import { ref, computed } from "vue";
 import arrowSvg from "../../assets/svg/down-arrow.svg";
 
+const emit = defineEmits(["update:modelValue", "change-placeholder"]);
 const priorities = ["Low", "Medium", "High"] as const;
 const selectedPriority = ref<(typeof priorities)[number]>(priorities[2]);
```

Comment:
same here, what happens if the todo's priority initial value is not low?

Reply:
This will be fixed when implementing reading/writing from local storage